templates/rubocop.rb

```diff
@@ -48,4 +48,6 @@
 end
 
 run "bundle binstubs rubocop"
-run "bin/rubocop -A"
+
+files_count = Dir["app/**/*", "lib/**/*", "spec/**/*", "test/**/*", "config/**/*"].length
```

Comment:
What do you think about counting only .rb files? Also does rubocop scan .erb files?

Reply:
What gives us a point number of files? For the argument, the main thing is to specify a number that exceeds the number of files that will be checked.